src/analyze/analyze.c

```diff
@@ -1592,6 +1594,47 @@ static int dump_exit_status(int argc, char *argv[], void *userdata) {
         return table_print(table, NULL);
 }
 
+static int dump_capabilities(int argc, char *argv[], void *userdata) {
+        _cleanup_(table_unrefp) Table *table = NULL;
+        int r;
+
+        table = table_new("name", "number");
+        if (!table)
+                return log_oom();
+
+        (void) table_set_align_percent(table, table_get_cell(table, 0, 1), 100);
+
+        if (strv_isempty(strv_skip(argv, 1)))
+                for (unsigned c = 0; c <= MAX((unsigned) CAP_LAST_CAP, cap_last_cap()); c++) {
+                        r = table_add_many(table,
+                                           TABLE_STRING, capability_to_name(c) ?: "cap_???",
+                                           TABLE_UINT, c);
+                        if (r < 0)
+                                return table_log_add_error(r);
+                }
+        else {
+                for (int i = 1; i < argc; i++) {
+                        int c;
+
+                        c = capability_from_name(argv[i]);
```

Comment:
If a large numeric value is specified, and both kernel and systemd do not know the corresponding capability, then it still shows `cap_???`. I think we need to check the value is less or equal to `MAX(CAP_LAST_CAP, cap_last_cap())`.

Reply:
Currently, 40 (cap_checkpoint_restore) is the largest known capability. But, if, e.g. `systemd-analyze capability 60`, then it shows `cap_??? 60`. If this is acceptable, then man page needs to be updated. Currently, it is documented that `cap_???` is shown only when the capability is known by kernel but not by systemd.

Reply:
will fix